assembly/Quat.ts

```diff
@@ -213,10 +216,22 @@ export class Quat {
         var m22 = m_.m10;
 
         // Remove the scale from the matrix
-        var lx: f32 = 1.0 / Mathf.sqrt(m00 * m00 + m01 * m01 + m02 * m02);
-        var ly: f32 = 1.0 / Mathf.sqrt(m10 * m10 + m11 * m11 + m12 * m12);
-        var lz: f32 = 1.0 / Mathf.sqrt(m20 * m20 + m21 * m21 + m22 * m22);
-
+        //var lx: f32 = 1.0 / Mathf.sqrt(m00 * m00 + m01 * m01 + m02 * m02);
+        //var ly: f32 = 1.0 / Mathf.sqrt(m10 * m10 + m11 * m11 + m12 * m12);
+        //var lz: f32 = 1.0 / Mathf.sqrt(m20 * m20 + m21 * m21 + m22 * m22);
+        lx = m00 * m00 + m01 * m01 + m02 * m02;
+        if (lx === 0)
```

Comment:
```suggestion
        if (Mathf.abs(lx) <= 1e-6)
```

Reply:
Any other opinions on this one? I am not sure

Reply:
Or `abs(x) <= 7e-46` https://webassembly.studio/?f=nw1g86emsnl